docs/Dialects/LTL.md

```diff
@@ -175,6 +200,9 @@ The `ltl.delay` sequence operation represents various shorthands for the *next*/
 | `ltl.delay %a, 0`    | **F**a                      |
 | `ltl.delay %a, 2`    | **XXF**a                    |
 
+### Until, Always and Eventually
+
+`ltl.until` and `ltl.always` are *weak*, meaning the property will hold even if there isn't enough clock to evaluate the property. `ltl.eventually` is *strong*, where `ltl.eventually %p` means `p` must hold at some point in the trace.
```

Comment:
Rewrite: """ltl.until` and `ltl.always` are *weak*, meaning the property will hold even if the trace does not contain clock cycles to evaluate the property."""    
  
Maybe it should also be noted that the default case in LTL would be `ltl.always`.

Reply:
I am not sure what this "default case" means. Does it mean that `assert(p)` will check `always p` by default, or will it check `p` every clock?

I notice that SVA has a global `assert` and an `initial assert`. global `assert(s_eventually p)` checks `s_eventually p` every clock, and `initial assert(s_eventually p)` only checks once. It seems we don't have a corresponding feature.

Reply:
The current way the ltl stuff is handled in CIRCT (from what I've seen through my experience playing around with it) is that is considers that an assertProperty will by default be an assert always. That is also how it's being treated in the ltl lowering to core.